src/index.html

```diff
@@ -10,8 +10,185 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
     <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form action="ttps://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="group">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label>
+          Surname:
+            <input
+          autocomplete="off"
+          type="text"
+          name="surname"
+          >
```

Comment:
```suggestion
            <input
              autocomplete="off"
              type="text"
              name="surname"
            >
```
follow the [styleguide](https://mate-academy.github.io/style-guides/htmlcss.html)(everywhere) 😉